conans/model/info.py

```diff
@@ -530,9 +517,10 @@ def serialize_min(self):
         """
         This info will be shown in search results.
         """
+        # Lets keep returning the legacy "full_requires" just in case some client uses it
         conan_info_json = {"settings": dict(self.settings.serialize()),
                            "options": dict(self.options.serialize())["options"],
-                           "full_requires": self.full_requires.serialize()
+                           "requires": self.requires.serialize()
```

Comment:
Not an issue for compatibility, the client has had a ``requires->full_requires`` fallback for a long time.